docs/development.md

```diff
@@ -225,106 +225,6 @@ is built as part of the LLVM project along with MLIR.
 We mount a ccache and pip cache inside the docker container to speed up iterative builds. Iterative
 builds should be as fast as running without docker.
 
-### In-Tree builds
```

Comment:
One small request: Could you paste the stuff being removed from `docs/development.md` into `build_linux_packages.sh` (as comments) for future reference? Some of this is useful info and carrying it forward makes sense in the script (not in the readme).